app/coffee/Features/Labels/LabelsController.coffee

```diff
@@ -0,0 +1,26 @@
+EditorRealTimeController = require "../Editor/EditorRealTimeController"
+LabelsHandler = require './LabelsHandler'
+logger = require 'logger-sharelatex'
+
+
+module.exports = LabelsController =
+
+	getAllLabels: (req, res, next) ->
+		project_id = req.params.Project_id
```

Comment:
We should default to lower case `project_id` for new methods. We'll get everything else migrated eventually!